docs/DeveloperGuide.md

```diff
@@ -294,12 +294,12 @@ _{more aspects and alternatives to be added}_
 ### \[Proposed\] Add Delete Task Feature (Ivor)
 #### Proposed Implementation
 The proposed delete task feature is facilitated by `DeleteTaskCommand`. It extends `Command` and make use of a new model `TaskList` and `Task`.
-The `TaskList` model consists of an `ArrayList<Task>` to store the `Task`. The `DeketeTaskCommand` also has a `DeleteTaskCommandParser`
+The `TaskList` model consists of an `ArrayList<Task>` to store the `Task`. The `DeleteTaskCommand` also has a `DeleteTaskCommandParser`
```

Comment:
Thank you for catching this error! 👍

Reply:
we have serious problems with spelling 😢